package.json

```diff
@@ -31,18 +31,18 @@
   },
   "homepage": "https://github.com/neverendingqs/serverless-dotenv-plugin#readme",
   "dependencies": {
-    "chalk": "^4.1.0",
-    "dotenv": "^8.2.0",
+    "chalk": "^4.1.2",
+    "dotenv": "^10.0.0",
```

Comment:
https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md

Looks tame enough. Biggest change I see is dropping support for Node 8, which doesn't affect us.